app/src/main/java/com/example/tung/demoretrofit/BindingUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.example.tung.demoretrofit;
+
+import android.databinding.BindingAdapter;
+import android.support.v7.widget.RecyclerView;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.example.tung.demoretrofit.data.model.Cast;
+import com.example.tung.demoretrofit.data.model.SubMovie;
+import com.example.tung.demoretrofit.sceen.HomeAdapter;
+
+import java.util.List;
+
+public class BindingUtils {
+
+    @BindingAdapter("imageView")
+    public static void loadImage(ImageView view, String url) {
+        Glide.with(view.getContext()).load(url).into(view);
+    }
+
+    @BindingAdapter("bindMovie")
+    public static void setMovieRecyclerView(RecyclerView recyclerView, List<SubMovie> movies) {
+        HomeAdapter adapter = (HomeAdapter) recyclerView.getAdapter();
```

Comment:
e nên xử lý tạo adapter trong code và chỉ đưa adapter vào đây sử dụng là đủ rồi e. như vậy e có thể tái sử dụng nhiều lần thay vì phải set cứng adapterHome như thế này
```suggestion
       @BindingAdapter("bindMovie")
     public static void setMovieRecyclerView(RecyclerView recyclerView, RecyclerView.Adapter adapter) {
        recyclerView.setAdapter(adapter)
         }
     }
```